### PR TITLE
[libspatialindex] Update to 1.9.3

### PR DESCRIPTION
**Pull request by m-kuhn**

### CODE REVIEW

File: ports/libspatialindex/portfile.cmake [hunk 1]
```diff
@@ -22,4 +20,4 @@ file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
 
 # Handle copyright
 file(COPY ${SOURCE_PATH}/COPYING DESTINATION ${CURRENT_PACKAGES_DIR}/share/libspatialindex)
-file(RENAME ${CURRENT_PACKAGES_DIR}/share/libspatialindex/COPYING ${CURRENT_PACKAGES_DIR}/share/libspatialindex/copyright)
\ No newline at end of file
+file(RENAME ${CURRENT_PACKAGES_DIR}/share/libspatialindex/COPYING ${CURRENT_PACKAGES_DIR}/share/libspatialindex/copyright)
```
Comment (JonLiu1993) [inline ports/libspatialindex/portfile.cmake]: @m-kuhn ,Thanks for your pr,please use "file(INSTALL ${SOURCE_PATH}/COPYING DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)" to replace the deprecated function

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]
When updating the port, we need to update the CONTROL file and portfile.cmake

**Comment by m-kuhn** [comment]
> When updating the port, we need to update the CONTROL file and portfile.cmake

Anything I can do?

**Comment by JonLiu1993** [comment]
> > When updating the port, we need to update the CONTROL file and portfile.cmake
> 
> Anything I can do?

Please update CONTROL file

**Comment by m-kuhn** [comment]
Update with what exactly?

**Comment by JonLiu1993** [comment]
> Update with what exactly?

Please run the command "./vcpkg x-add-version libspatialindex --overwrite-version" and submit again

**Comment by m-kuhn** [comment]
Thanks for the hint, PR updated

**Comment by dg0yt** [comment]
Hi Matthias,

you must first commit all changes in port/*, then do the x-add-version stuff in a separate commit. 

Best regards, Kai

**Comment by JonLiu1993** [comment]
@m-kuhn , @dg0yt gave you a good suggestion

**Comment by m-kuhn** [comment]
Thanks Kai, I hope ci is happy now.

**Comment by dg0yt** [comment]
> Thanks Kai, I hope ci is happy now.

Not yet. A single commit is never enough.
Re-run `vcpkg x-add-version --overwrite-version libspatialindex`. Then add the changes in a **separate** commit.
The point is that this adds a reference to a particular git tree element (`versions/*` refering to a particular state of `ports/*`). If you squash the changes, you would need to know the resulting reference in advance, but this is impossible.

Normally I would also suggest to first run `vcpkg format-manifest ports/libspatialindex/CONTROL` in order to convert away from the deprecated CONTROL format, but I don't want to ask too much at the moment ;-)

**Comment by dg0yt** [comment]
Please add commits as requested, and push normally.  Force-pushing a single commit is going to fail. The last commit must be the change from vcpkg x-add-version.

**Comment by m-kuhn** [comment]
I think the culprit here is static builds

> DLLs should not be present in a static build, but the following DLLs were found:
> 
>     D:/packages/libspatialindex_x64-windows-static-md/bin/spatialindex-64.dll
>     D:/packages/libspatialindex_x64-windows-static-md/bin/spatialindex_c-64.dll
>     D:/packages/libspatialindex_x64-windows-static-md/debug/bin/spatialindex-64d.dll
>     D:/packages/libspatialindex_x64-windows-static-md/debug/bin/spatialindex_c-64d.dll
> 
> There should be no bin\ directory in a static build, but D:\packages\libspatialindex_x64-windows-static-md\bin is present.
> There should be no debug\bin\ directory in a static build, but D:\packages\libspatialindex_x64-windows-static-md\debug\bin is present.
> If the creation of bin\ and/or debug\bin\ cannot be disabled, use this in the portfile to remove them
>

**Comment by JonLiu1993** [comment]
@m-kuhn ,Thanks for your pr, Duplicate of #17506

**Pull request closed without merging**